### ISSUE TITLE
Joints are not deleted when taps "delete" button while selecting joints in timeline.

### ISSUE THREAD

**Issue by Kanoot** [spritebuilder/SpriteBuilder#919]
1. Launch application.
2. Create a new project.
3. Drag and Drop 2 Physics Joint under Joint.
4. Select Joint in timeline.
5. Click delete.
   
   -- Result: PhysicsJoints under the Joint select down are not deleted and PhysicsJoints become empty.

Video: https://drive.google.com/file/d/0B5KeRaV3CWUaTHVQZGdTWFBmTjA/edit?usp=sharing

This happened 3/3 times.
Build: SpriteBuilder 1.3 Beta 07

Mac OS X: 10.9.2.
XCode: 5.0.

**Comment by khanbhai89** [comment]
This issue is still occurring on SpriteBuilderAdHoc-sb-1.3.0-rc.1-d49d1950d11b5a27d61b6234761197522d03875a_d6d92a9f14495012

Mac OS X: 10.9.5
XCode: 6.0

**Comment by khanbhai89** [comment]
This issue is still occurring on SpriteBuilder-1.3.0-rc.1.zip on Mac OS X: 10.9.5. and XCode: 6.0.

**Comment by Kanoot** [comment]
This issue is reoccurring.

SpriteBuilder-1.3.4-rc.1

Device A:
Mac OS X: 10.9.5
XCode: 6.0

**Comment by cocojoe** [comment]
Not sure if hitting delete on the top level joint should do anything, as hitting delete on the top level `CCNode` will not delete all children.

**Comment by Kanoot** [comment]
@cocojoe -- Hit delete on top of the joint not on CCNode. Sorry for misunderstanding. Screenshot is attached.

![screen shot 2014-11-11 at 10 10 02 pm](https://cloud.githubusercontent.com/assets/7566802/4997343/ba16ca7a-69ef-11e4-80f4-b101f4c6d437.png)

**Comment by cocojoe** [comment]
Yeah I understood what you ment, thanks for the screenshot.  I was asking the question in general to the other developers if allowing this behaviour is actually a mistake (which I think it is).  Thanks

**Comment by Kanoot** [comment]
This issue is reoccurring.

Build: SpriteBuilder-1.3.5-rc.2

MAC OS: 10.9.5
XCode: 6.0

**Comment by khanbhai89** [comment]
This issue is still occurring on SpriteBuilder-1.3.6.rc.1.

Mac OS X: 10.9.5.
XCode: 6.0.

**Comment by Kanoot** [comment]
This issue is still not fixed.

Build: SpriteBuilder-1.3.6.rc.1

Mac OS X: 10.9.5.
XCode: 6.0.
Device: Nexus 4/4.3

**Comment by sana20** [comment]
This issue is still occurring on SB Version: 1.4.0-beta.02
Mac OS X: 10.10.1.
XCode: 6.0.

**Comment by khanbhai89** [comment]
This issue is still occurring on SpriteBuilder-1.4.0-rc.0.zip

Mac OS X:10.9.5
XCode: 6.0.

**Comment by sana20** [comment]
This issue is re-occurring on SpriteBuilder-1.4.8-rc.0 with SBAndroid Indie (SBAndroidPluginIndie-sbandroid-indie-704bb7c4b3507f7e).

Mac OS X: 10.10.1
XCode: 6.1.
Device: Note 2/4.1.1.

**Comment by Shahidwerplay** [comment]
This issue is still occurring on SpriteBuilder-1.4.8-rc.0 with SBAndroidPluginStarter-9ac08c84.
MAC OS X: 10.10.
XCode: 6.1.

**Comment by khanbhai89** [comment]
This issue is still occurring on SpriteBuilder-1.4.9.-rc.0 on Mac OS X:10.9.5.

**Comment by NoorFatima** [comment]
This issue is re-occurring.

Mac OS X: 10.9.5
XCode: 6.1
SB Version: 2.0.0
SB Revision: 4fd68fb

**Comment by khanbhai89** [comment]
This issue is reoccurring.

Mac OS X 10.10.0
SB Version latest develop branch (2.0)
SB Revision 4fd68fb 
XCode 6.1
